orb/commands/github_release.yml

```diff
@@ -0,0 +1,58 @@
+description: >
```

Comment:
Remarkably similar to the `release.sh` repo

Reply:
>Mostly stolen from here: https://raw.githubusercontent.com/FairwindsOps/release.sh/v0.0.2/release.

From the description of the PR